threeML/plugins/EventListLike.py

```diff
@@ -73,11 +71,55 @@ def __init__(self, name, event_list, background_selections, source_intervals, rs
 
         self._verbose = verbose
 
+        try:
+
+            OGIPLike.__init__(self,
+                              name,
+                              pha_file=self._observed_pha,
+                              bak_file=self._bkg_pha,
+                              rsp_file=rsp_file,
+                              verbose=verbose)
+
+        except(NegativeBackground):
```

Comment:
If I understand well what is going on here, I am not sure this is right.

A negative background does not necessarily mean that the polynomial grade is too high... I'd rather throw an exception and advise to change the background off-pulse intervals.